libraries/hotstuff/test/test_hotstuff_state.cpp

```diff
@@ -0,0 +1,163 @@
+#include <stdio.h>
+
+#include <boost/test/unit_test.hpp>
+#include <eosio/chain/types.hpp>
+
+#include <fc/io/cfile.hpp>
+
+#include <fc/exception/exception.hpp>
+#include <eosio/hotstuff/qc_chain.hpp>
+
+#include <fc/crypto/bls_signature.hpp>
+
+using std::cout;
+
+BOOST_AUTO_TEST_SUITE(test_hotstuff_state)
+
+const std::string file_path_1("temp_hs_safety");
```

Comment:
You can use `fc::temp_cfile` or `fc::temp_directory` to create a temp file such that it will be automatically removed from test machine when test is finished.

Reply:
`state_db_manager` (from `qc_chain.hpp`) has methods to operate on `fc::cfile`, doesn't seem that `fc::temp_cfile` is interchangeable with it. Also, just passing a string file name to it lets it deal with the `cfile` detail.